### PR TITLE
Get slice helm

### PR DESCRIPTION
**Pull request by Guseyn**
Get slice helm: regarding #1

### CODE REVIEW

**Comment by Guseyn** [comment]
@g4s8 seems like zero-bot is not reacting at all, could you plz make a progress here?

File: src/main/java/com/artipie/helm/HelmSlice.java [hunk 1]
```diff
@@ -39,15 +41,33 @@
  * @since 0.1
  */
 public final class HelmSlice implements Slice {
+
+    /**
+     * Route.
+     *
+     * For GET requests we just need to serve files.
+     */
+    private final SliceRoute route;
```
Comment (g4s8) [inline src/main/java/com/artipie/helm/HelmSlice.java:50]: @Guseyn you can extend from `Slice.Wrap` which contains decorating logic

File: src/main/java/com/artipie/helm/HelmSlice.java [hunk 2]
```diff
@@ -39,15 +41,33 @@
...
+
+    /**
+     * Ctor.
+     *
+     * @param storage Storage for Helm.
+     */
+    public HelmSlice(final Storage storage) {
+        this.route = new SliceRoute(
```
Comment (g4s8) [inline src/main/java/com/artipie/helm/HelmSlice.java:58]: @Guseyn and call `super(new SliceRoute(...))` here

File: src/test/java/com/artipie/helm/HelmGetITCase.java [hunk 3]
```diff
@@ -35,28 +39,34 @@
...
 
     @Test
-    public void notImplemented() throws IOException {
+    void notImplemented() throws IOException {
         final Vertx vertx = Vertx.vertx();
+        final Storage storage = new InMemoryStorage();
+        final String path = "/charts/index.yml";
+        storage.save(
+            new KeyFromPath(path),
+            new Content.From("content".getBytes())
```
Comment (g4s8) [inline src/test/java/com/artipie/helm/HelmGetITCase.java:55]: @Guseyn please use charset encoding explicitly to avoid platform dependent builds

**Pull request closed without merging**